flexmeasures/cli/data_show.py

```diff
@@ -21,7 +22,7 @@ def list_accounts():
     """
     List all accounts on this FlexMeasures instance.
     """
-    accounts = Account.query.all()
+    accounts = Account.query.order_by(Account.id).all()
```

Comment:
A general comment, really: I believe a common use for these commands is to look up an id, given knowledge of a name. Therefore, it would be more practical to order by name instead of by id.

At least for accounts, users, both roles and data sources.

I would suggest making that the default, and to introduce a CLI option for each command to order by id if needed.

Reply:
I agree to sort by name when possible. I don't think we need to add the --order-by-id option today.

Reply:
Okay.

Reply:
I'm going with ordering everything by name and we will learn what we need from using. Your suggestion is very likely an improvement.